File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex3.c
#include <stdio.h>

main()
{
    //vars
    int n1, n2, res;

    printf("\t\t\tDivisao entre dois numeros\n\n");

    //Entradas
    printf("Insira o 1o numero: ");
    scanf("%d",&n1);

    printf("Insira o 2o numero (diferente de zero): ");
    scanf("%i",&n2);

    //Processamento
    res = n1 / n2;

    //Saída
    printf("%i / %d = %i", n1, n2, res);


}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex3.java
//Importando Scanner
import java.util.Scanner;

public class Ex3{
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	//vars
	float n1, n2, res;
	
	System.out.println("Divisão entre dois números\n\n");
	
	//Entradas
	System.out.print("Insira o 1° número: ");
	n1 = ent.nextFloat();
	
	System.out.print("Insira o 2° número (diferente de zero): ");
	n2 = ent.nextFloat();
	
	//Processamento
	res = n1 / n2;
	
	//Saída
	System.out.println(n1 + " / " + n2 + " = " + res);
		
	}
}